### PR TITLE
Add contribution guidelines

### PR DESCRIPTION
**Pull request by jackye1995**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,15 +26,59 @@ It's built with [Hugo](https://gohugo.io/) and hosted at https://iceberg.apache.
...
+To start the documentation site locally, clone this repository and run the following.
+```shell
+git clone git@github.com:apache/iceberg-docs.git
+git submodule update --init
+cd docs && hugo serve
+```
+
+## Viewing Latest Website
+
+If you would like to see how the latest website looks based on the documentation in the Iceberg repository, you can copy docs to this repository by:
```
Comment (jackye1995) [inline README.md:63]: need to refine this workflow a bit, does the `docs/content/docs` path contains all the documentations user might modify?

Comment (jackye1995) [reply README.md:63]: Looks like there is another part in the `landing-page/common`

Comment (jackye1995) [reply README.md:63]: okay so I assume the iceberg repo would have a `documentation` folder and `common` and `docs` folder inside mapping to the 2 sites here.

File: README.md [hunk 2]
```diff
@@ -26,15 +26,62 @@ It's built with [Hugo](https://gohugo.io/) and hosted at https://iceberg.apache.
 
 The Iceberg documentation site is actually constructed from two hugo sites. The first, is the landing page. The second site, 
 is the documentation site which contains the full Iceberg documentation, including the javadoc. The landing page and
-documentation sites are completely self-contained in the `./landing-page` and `./docs` directories, respectively.
+documentation sites are completely self-contained in the `./landing-page` and `./docs` directories, respectively. The Javadocs are in the `./javadoc` directory.
 
-# Landing Page Deployment
```
Comment (jackye1995) [inline README.md:31]: I moved up contribution guide because readers are most likely interested in that

File: README.md [hunk 3]
```diff
@@ -26,15 +26,62 @@ It's built with [Hugo](https://gohugo.io/) and hosted at https://iceberg.apache.
...
+# How to Contribute
+
+## Submitting Pull Requests
+
+All pull requests to update the latest documentations should be submitted to the [Iceberg repository](https://github.com/apache/iceberg).
+When Iceberg publishes a new release, the release manager copies all the documentation to this repository and cut a new version branch.
```
Comment (samredai) [inline README.md:36]: s/cut/cuts

File: README.md [hunk 4]
```diff
@@ -26,15 +26,62 @@ It's built with [Hugo](https://gohugo.io/) and hosted at https://iceberg.apache.
...
+
+Changes to the landing page should be submitted to this repository against the `main` branch.
```
Comment (samredai) [inline README.md:38]: The landing-page site includes the common markdown files as well (releases, spec, community, etc). We should keep those in `apache/iceberg` too and have PRs opened there, right? That would be simpler--all docs PRs are opened in `apache/iceberg` unless you're hotfixing a previous version in which case you open the PR against that particular version's branch in `apache/iceberg-docs`.

File: README.md [hunk 5]
```diff
@@ -26,15 +26,62 @@ It's built with [Hugo](https://gohugo.io/) and hosted at https://iceberg.apache.
...
+
+Changes to the documentation of old Iceberg versions should be submitted to this repository against the specific version branch.
+
+## Reporting Issues
+
+All issues related to documentation and website should still be submitted to the [Iceberg repository](https://github.com/apache/iceberg).
```
Comment (samredai) [inline README.md:44]: s/to documentation and website/to the documentation website

File: README.md [hunk 6]
```diff
@@ -26,15 +26,62 @@ It's built with [Hugo](https://gohugo.io/) and hosted at https://iceberg.apache.
...
+
+## Running Locally
+
+To start the landing page site locally, clone this repository and run the following.
```
Comment (samredai) [inline README.md:48]: nit: the code block includes the clone so maybe just say "To start the landing page site locally, run the following."

File: README.md [hunk 7]
```diff
@@ -26,15 +26,62 @@ It's built with [Hugo](https://gohugo.io/) and hosted at https://iceberg.apache.
...
+```shell
+git clone git@github.com:apache/iceberg-docs.git
+cd landing-page && hugo serve
+```
+
+To start the documentation site locally, clone this repository and run the following.
```
Comment (samredai) [inline README.md:54]: nit: same comment about the clone already being included in the code block.

File: README.md (hunk 1 above)
Comment (samredai) [reply README.md:63]: Yep, exactly right. In `apache/iceberg` we would have two directories that are intuitively named like `documentation/common` and `documentation/versioned` and they would hold the contents in [docs/content/docs](https://github.com/apache/iceberg-docs/tree/main/docs/content/docs) and [landing-page/content/common](https://github.com/apache/iceberg-docs/tree/main/landing-page/content/common), respectively. The release manager would then just do a full hard-copy over of those two directories (might be worth mentioning that it should be a full copy, meaning deleted files are removed in the destinations as well).

**Review by samredai — APPROVED** [review]
A few comments and nits but overall LGTM. Thanks @jackye1995 for adding this!

File: README.md (hunk 1 above)
Comment (samredai) [reply README.md:63]: Once this PR and PR #27 is merged, I'll go ahead and open the PR against `apache/iceberg` to add those `documentation/common` and `documentation/versioned` directories.

File: README.md (hunk 4 above)
Comment (jackye1995) [reply README.md:38]: yes agree, forgot to update this sentence after updating the scripts

File: README.md (hunk 1 above)
Comment (jackye1995) [reply README.md:63]: got it, let me update based on this info

File: README.md (hunk 6 above)
Comment (jackye1995) [reply README.md:48]: I will move the `git clone` part to another line, let me know what you think

**Pull request merged**